src/Windows.ShellScalingApi.Desktop/ShellScalingApi.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <summary>
+    /// Exported functions from the ShellScalingApi.h header.
+    /// </summary>
+    public partial class ShellScalingApi
```

Comment:
If ShellScalingApi is only for header file declarations, do we even need a test assembly?